packages/common/lib/types/StateManager.types.ts

```diff
@@ -4,6 +4,11 @@ export interface CredentialOfferState {
   clientId?: string;
   credentialOffer: CredentialOfferPayloadV1_0_11;
   createdOn: number;
+  'pre-authorized_code': string;
```

Comment:
Why is this stored separately? This is part of the Grant in the credentialOffer

Reply:
Fixed!